src/js/msp/MSPHelper.js

```diff
@@ -904,6 +906,16 @@ MspHelper.prototype.process_data = function(dataHandler) {
                 FC.TELEMETRY_CONFIG.telemetry_inverted = data.readU8();
                 FC.TELEMETRY_CONFIG.telemetry_halfduplex = data.readU8();
                 FC.TELEMETRY_CONFIG.telemetry_sensors = data.readU32();
+                if (data.remaining() === 6 + self.CRSF_TELEMETRY_SENSOR_LENGTH) {
```

Comment:
The MSP api has been changed to 12.7 for the 2.1.x release.
Can you check the API version here rather than the size.
```
if (semver.gte(FC.CONFIG.apiVersion, '12.7.0')) {
```

Reply:
Rebased master to include this change.